simple/single-script/wso2am-latest.sh

```diff
@@ -3435,12 +3435,12 @@ function deploy(){
     create_yaml
 
     # replace necessary variables
-    sed -i '' 's/"$ns.k8s.&.wso2.apim"/'$namespace'/g' $k8s_obj_file
-    sed -i '' 's/"$string.&.secret.auth.data"/'$secdata'/g' $k8s_obj_file
-    sed -i '' 's/"ip.node.k8s.&.wso2.apim"/'$NODE_IP'/g' $k8s_obj_file
-    sed -i '' 's/"$nodeport.k8s.&.1.wso2apim"/'$NP_1'/g' $k8s_obj_file
-    sed -i '' 's/"$nodeport.k8s.&.2.wso2apim"/'$NP_2'/g' $k8s_obj_file
-    sed -i '' 's/"$image.pull.@.wso2"/'$IMG_DEST'/g' $k8s_obj_file
+    sed -i.bak 's/"$ns.k8s.&.wso2.apim"/'$namespace'/g' $k8s_obj_file
```

Comment:
Here the Sed command creates backup files. Isn't it better to remove those? WDYT?

Reply:
Do you mean after creating use the rm command to remove the .bak file?

Reply:
yes